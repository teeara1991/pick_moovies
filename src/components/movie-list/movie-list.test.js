import React from 'react';
import renderer from 'react-test-renderer';
import MovieList from '../movie-list/movie-list.jsx';

it(`MovieList correctly renders after relaunch`, () => {
  const tree = renderer
      .create(<MovieList
        films={[
          {
            genre: `fantasy`,
            title: `Harry Potter`,
            logo: `https://st.kp.yandex.net/images/film_iphone/iphone360_689.jpg`,
            links: ``,
          },
          {
            genre: `fantasy`,
            title: `Harry Potter2`,
            logo: `https://upload.wikimedia.org/wikipedia/ru/thumb/0/0b/Harry_Potter_and_the_Chamber_of_Secrets_%E2%80%94_movie.jpg/219px-Harry_Potter_and_the_Chamber_of_Secrets_%E2%80%94_movie.jpg`,
            links: ``,
          }]
        }
      />)
      .toJSON();
  expect(tree).toMatchSnapshot();
});
