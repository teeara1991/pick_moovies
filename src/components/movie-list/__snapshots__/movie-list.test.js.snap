// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MovieList correctly renders after relaunch 1`] = `
Array [
  <article
    className="small-movie-card catalog__movies-card "
    onMouseOver={[Function]}
  >
    <button
      className="small-movie-card__play-btn"
      onClick={[Function]}
      type="button"
    >
      Play
    </button>
    <div
      className="small-movie-card__image"
    >
      <img
        alt="Harry Potter"
        height="175"
        src="https://st.kp.yandex.net/images/film_iphone/iphone360_689.jpg"
        width="280"
      />
    </div>
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
        href=""
      >
        Harry Potter
      </a>
    </h3>
  </article>,
  <article
    className="small-movie-card catalog__movies-card "
    onMouseOver={[Function]}
  >
    <button
      className="small-movie-card__play-btn"
      onClick={[Function]}
      type="button"
    >
      Play
    </button>
    <div
      className="small-movie-card__image"
    >
      <img
        alt="Harry Potter2"
        height="175"
        src="https://upload.wikimedia.org/wikipedia/ru/thumb/0/0b/Harry_Potter_and_the_Chamber_of_Secrets_%E2%80%94_movie.jpg/219px-Harry_Potter_and_the_Chamber_of_Secrets_%E2%80%94_movie.jpg"
        width="280"
      />
    </div>
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
        href=""
      >
        Harry Potter2
      </a>
    </h3>
  </article>,
]
`;
